akka-actor/src/main/resources/reference.conf

```diff
@@ -276,6 +276,9 @@ akka {
 
         # Max number of threads to cap factor-based parallelism number to
         parallelism-max = 64
+
+        # Enable establishing local first-in-first-out scheduling mode for forked tasks that are never joined.
+        async-mode = on
```

Comment:
default is `false`, so we might want to keep this default `off` for 2.3 to not change behavior?